### DISCUSSION TITLE
Github Organization

### DISCUSSION THREAD

**Discussion by ankushkatiyar** [community/community#23785] [New to GitHub]
<p>Hi I have a pro account of GitHub and I want to use Github organization, so I need to pay any extra charges for the created organization or not or create teams and repositories? </p>
<p>Thanks</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>HI <a class="mention" href="/ankushkatiyar">@ankushkatiyar</a>, thank you for being here and my apologies for the delayed response. Personal accounts and organization accounts are entirely separate.</p>
<p>On April 14 we announced that our free plans come with private repositories and unlimited users, so you may just want to create a free organization account first:</p>
<p><a href="https://help.github.com/en/github/getting-started-with-github/faq-about-changes-to-githubs-plans" class="onebox" target="_blank">https://help.github.com/en/github/getting-started-with-github/faq-about-changes-to-githubs-plans</a></p>
<p><a href="https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/creating-a-new-organization-from-scratch" class="onebox" target="_blank">https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/creating-a-new-organization-from-scratch</a></p>
<p>Then if you would like the additional paid features, you can upgrade to the Team plan which costs $4 per user/month for any user account that has access to a private repository:</p>
<p><a href="https://help.github.com/en/articles/githubs-products" class="onebox" target="_blank">https://help.github.com/en/articles/githubs-products</a></p>
<p><a href="https://help.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/upgrading-your-github-subscription" class="onebox" target="_blank">https://help.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/upgrading-your-github-subscription</a></p>
<p>I hope this helps!</p>
